packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -64,29 +64,41 @@ class RecoveryPhoneHandler {
   getLocalizedMessage = async (
     request: AuthRequest,
     code: string,
-    type: 'setup' | 'signin'
+    type: 'setup' | 'signin' | 'setup-short' | 'signin-short'
   ) => {
-    const id =
-      type === 'setup'
-        ? 'recovery-phone-setup-sms-body'
-        : 'recovery-phone-signin-sms-body';
-
-    const fallbackMessage =
-      type === 'setup'
-        ? `${code} is your Mozilla verification code. Expires in 5 minutes.`
-        : `${code} is your Mozilla recovery code. Expires in 5 minutes.`;
+    const l10nTypeMap = {
+      setup: {
```

Comment:
I do really like the flexibility of this 👍🏽

Reply:
This is much nicer 👍